script.js

```diff
@@ -11,4 +11,137 @@ document.querySelectorAll(".nav-link").forEach(n => n.
+    <h2 class="title">${projects.title1}</h2>
+                        <ul class="highlights">
+                            <li>CANOPY</li>
+                            <li><img class="counter" src="Counter.png" alt="counter"></li>
+                            <li>Back End Dev</li>
+                            <li><img class="counter" src="Counter.png" alt="counter"></li>
+                            <li>2015</li>
+                        </ul>
+                        <p class="description">${projects.description}</p>
+                        <ul class="languages">
+                            <li class="stack">${projects.technologies[0]}</li>
+                            <li class="stack">${projects.technologies[1]}</li>
+                            <li class="stack">${projects.technologies[2]}</li>
+                        </ul>
+    `
+    elements.append(button)
+    divImg.append(img)
+    article.append(divImg, elements)
+    section.append(article)
+
+    button.addEventListener("click", () => {
+        popup.style.display = 'flex'
+        section.style.filter = 'blur(5px)'
+        document.body.style.overflow = 'hidden'
+        popup.innerHTML = `
+    <div class="popup-header">
+                    <h2 class="popup-title">${projects.title1}</h2>
+                    <div class="cancel-icon"><img src="./ICONS/cancel-icon.png"></div>
+                </div>
+                <ul class="popup-highlights">
+                    <li>CANOPY</li>
+                    <li><img class="counter" src="Counter.png"></li>
+                    <li>Back End Dev</li>
+                    <li><img class="counter" src="Counter.png"></li>
+                    <li>2015</li>
+                </ul>
+                <img src="${projects.img2}">
+                <div class="popup-content">
+                    <div class="left-section">
+                        <p class="project-descritpion">${projects.description2}</p>
+                    </div>
+                    <div class="right-section">
+                        <ul class="stacks popup">
+                            <li class="stack">Html</li>
+                            <li class="stack">Css</li>
+                            <li class="stack">Javascript</li>
+                        </ul>
+                        <div class="popup-btns">
+                            <a href="${projects.liveDemo}" class="popup-btn" target="_blank">See live <img src="./ICONS/live.png"></a>
+                            <a href="#" class="popup-btn" target="_blank">See source<img src="./ICONS/github.png"></a>
+                        </div>
+                    </div>
+                </div>
+    `
+        const cancel = document.querySelector(".cancel-icon")
+        cancel.addEventListener("click", () => {
+            popup.style.display = 'none'
+            section.style.filter = 'none'
+            document.body.style.overflow = 'initial'
+        })
+    })
+})
+// const body = document.querySelector("body");
+
```

Comment:
- [ ] [OPTIONAL] Kindly delete any commented code, so that your code looks more professional.